### DISCUSSION TITLE
Release v1.1.1 / 5.56.1 (pre-release)

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#1932] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This is an experimental release it changes how sandboxie marks deleted host files as deleted, instead of creating dummy entries it uses an internal data structure to keep the stat of changes to the host file system. With the new mechanics it is possible to rename folders and keys properly.
This new behavior needs to be enabled by specifying UseFileDeleteV2=y sand UseRegDeleteV2=y in later builds this will be enabled by default, hence please test this pre release build and report back if there are any issues.

EDIT: please note due to a bug in the current build the UseFileDeleteV2=y sand UseRegDeleteV2=y requires also SeparateUserFolders=y this will be fixed in the next build.

This build also adds a new UI functionality, sandman.exe now monitors the size of sandboxes and reports it in the info column, this functionality needs to be enabled in the global settings.

Last but not least this build reduces the cpu usage of sandman.exe a lot.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

# ChangeLog

### Added
- compatibility templates can now be viewed from the settings window [#1891](https://github.com/sandboxie-plus/Sandboxie/issues/1891)
- the refresh command is now bound to F5 [#1885](https://github.com/sandboxie-plus/Sandboxie/issues/1885)
- added more first start wizard options
- added option to permanently disable immediate recovery for any given box when it opens [#1478](https://github.com/sandboxie-plus/Sandboxie/issues/1478)
- double click on the path column now opens the box root in explorer [#1924](https://github.com/sandboxie-plus/Sandboxie/issues/1924)

### Changed
- changed Move Box behaviour [#1879](https://github.com/sandboxie-plus/Sandboxie/issues/1879)
- improved implementation of the PreferExternalManifest option
- Win32k hooks are now by default only used for Edge and Chromium apps as they cause issues with other software [#1902](https://github.com/sandboxie-plus/Sandboxie/issues/1902) [#1912](https://github.com/sandboxie-plus/Sandboxie/issues/1912) [#1897](https://github.com/sandboxie-plus/Sandboxie/issues/1897)
- "AllowBoxedJobs=y" is now the default behaviour

### Fixed
- fixed Edge issue with Windows 11 after KB5014019
- fixed issues with the new delete V2 mechanism when using "SeparateUserFolders=y" [#1885](https://github.com/sandboxie-plus/Sandboxie/issues/1885)
- fixed credential issue [#1770](https://github.com/sandboxie-plus/Sandboxie/pull/1770)
- fixed force process priorities [#1883](https://github.com/sandboxie-plus/Sandboxie/issues/1883)
- fixed issues with the new delete V2 mechanism
- fixed issue with the Windows 11 menu on older Windows builds [1877](https://github.com/sandboxie-plus/Sandboxie/issues/1877)
- refresh now works without WatchBoxSize option [#1885](https://github.com/sandboxie-plus/Sandboxie/issues/1885)
- fixed crash issue with WatchBoxSize=true [#1885](https://github.com/sandboxie-plus/Sandboxie/issues/1885)
- fixed issue with recovery folder paths [#1840](https://github.com/sandboxie-plus/Sandboxie/issues/1840)
- fixed issues with Sbie desktop and wndStation affecting Acrobat Reader [#1863](https://github.com/sandboxie-plus/Sandboxie/issues/1863)
- fixed issues with box grouping [#1921](https://github.com/sandboxie-plus/Sandboxie/issues/1921) [#1920](https://github.com/sandboxie-plus/Sandboxie/issues/1920)
- fixed issues when changing language [#1914](https://github.com/sandboxie-plus/Sandboxie/issues/1914)
- fixed issue with BreakoutFolder [#1908](https://github.com/sandboxie-plus/Sandboxie/issues/1908)
- fixed issue with SbieDll.dll for x86 exception handling
- fixed issues with application-specific hives (RegLoadAppKey) affecting Visual Studio [#1576](https://github.com/sandboxie-plus/Sandboxie/issues/1576) [#1452](https://github.com/sandboxie-plus/Sandboxie/issues/1452)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/1.1.1'>Release v1.1.1 / 5.56.1</a>.</em>

**Comment by BilleBarrett** [comment]
Microsft edge just updated and Sandboxie-Plus doesn't work
Before updating Microsoft Edge
Version 100.0.1185.50 (Official Build) (64-bit)
Updated  Microsoft Edge
Version 102.0.1245.41 (Official Build) (64-bit)
I tried to see what was updated, 
and found this when I updated a windows 10 system windows 11
I checked my Microsft edge and it started an update, had to do it unsandboxed 
After the update, I just get a white screen

**Reply by DavidXanatos** [reply]
there is a new compatibiltiy tempalte called edge fix you need to enable it, also use build 1.1.2
last but not least disable AllowBoxedJobs=n

**Reply by BilleBarrett** [reply]
David, it was checked and I am using 1.1.2

**Reply by BilleBarrett** [reply]
Last but not least disable AllowBoxedJobs=n <<< where is this setting

**Reply by BilleBarrett** [reply]
Sorry I just found it  :-), that corrected it

**Reply by BilleBarrett** [reply]
Well Edge loaded bur after it open no webpages are loading, works great on Chrome

**Reply by BilleBarrett** [reply]
When I try to terminate msedge.exe I get Error Status 0xc000000b (The parameter is incorrect.)
There are 3 instances of msedge.exe showing in Sandboxie-Plus and the task manager show nothing running

**Reply by BilleBarrett** [reply]
I caused the problem with Edge not being deleted out of Sandboxie-Plus 
I checked the box for "Protect the sandbox from deleting or emptying"
Edge opens the info is there at the top and it asks me to restore, which I did 
But there is nothing displaced
I'm going to do a fresh install of Sandboxie-Plus with a new INI file, in case i screwed something up

**Reply by BilleBarrett** [reply]
After reinstalling Sandbox-Plus and setting  AllowBoxedJobs=n
Both opened OK
Sorry to bother you
Bill

**Reply by BilleBarrett** [reply]
Does it matter where you put the in the ini file
UseFileDeleteV2=y 
UseRegDeleteV2=y 
SeparateUserFolders=y
